src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -342,7 +342,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
         Private ReadOnly Property IInstance As IExpression Implements IInvocationExpression.Instance
             Get
-                Return Me.ReceiverOpt
+                If Me.Method.IsShared Then
```

Comment:
This is perfectly fine, but could also be coded as Return If(Me.Method.IsShared, Nothing, Me.ReceiveOpt) .